spec/dash-modal/view.coffee

```diff
@@ -234,3 +232,20 @@ describe 'DashModal.View', ->
       pressEscape()
 
       assertVisible(_m)
+
+  describe 'to prevent iframe issues', ->
```

Comment:
I would tweak the test descriptions a bit. Something like one of these:

```coffee
describe "Focusing on the view container", ->

  it "is allowed", ->
  
  it "moves focus to the view container when the modal is shown", ->


describe "Showing the modal", ->

  it "allows the view container to be focused", ->
  
  it "moves focus to the view container to prevent iframe issues", ->
```